packages/riipen-ui/src/components/Popover.jsx

```diff
@@ -228,6 +235,11 @@ class Popover extends React.Component {
       }
     }
 
+    // Handle fullWidth menu
+    if (fullWidth) {
+      left = 0;
```

Comment:
Added to end of function, if added to beginning, would need to wrap each of the changes to `left` throughout the function in an `if(!fullWidth){}`, since still need the `top` changes throughout the function to get `top` position of `Popover`

Feel this is cleaner / more easily changed

Reply:
that's perfectly reasonable